instructions.html

```diff
@@ -1,52 +1,63 @@
-                    <h1 class="section-header">Speedometer 3 Test Instructions</h1>
-                    <div class="section-content">
-                        <h2>General Instructions</h2>
-                        <ol>
-                            <li>Before running Speedometer:</li>
-                            <ul>
-                                <li>Close all other applications.</li>
-                                <li>Close other browser windows and tabs.</li>
-                                <li>Use a fresh browser profile and a single window.</li>
-                            </ul>
-                            <li>While running Speedometer:</li>
-                            <ul>
-                                <li>Keep the browser window in the foreground.</li>
-                                <li>Do not use any other applications while Speedometer runs.</li>
-                            </ul>
-                        </ol>
 
-                        <!-- TODO: fill in detailed instructions
-                        <h2>Detailed browser-specific instructions</h2>
-                        <h3>Chrome</h3>
-                        <h3>Edge</h3>
-                        <h3>Firefox</h3>
-                        <h3>Safari</h3>
-                    --></div>
-                    <div class="buttons section-footer">
-                        <div class="button-row">
-                            <a class="button" href="./" title="Show main section.">Home</a>
-                            <a class="button" href="about.html" title="Show about section.">About</a>
-                        </div>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=850" />
+    <title>Speedometer 3.0 Test Instructions</title>
+    <link rel="stylesheet" href="resources/main.css" />
+    <link rel="icon" href="resources/favicon.png" />
+</head>
+
+<body>
+    <main>
+        <section id="instructions" class="visible" data-title="Test Instructions">
+            <a href="./" class="logo">
+                <img srcset="resources/logo@2x.png 2x" src="resources/logo.png" alt="Speedometer" />
+                <div class="version">3.0</div>
+            </a>
+            <div class="section-grid">
+                <h1 class="section-header">Speedometer 3 General Instructions</h1>
+                <div class="section-content">
+                    <p>
+                        To get reasonably accurate results your device and browser need to be focused on the benchmark.
+                        Here are some basic instructions to allow that:
+                    </p>
+                    <ul class="compact">
+                        <li>
+                            Ensure you are running the latest browser version.
+                        </li>
+                        <li>
+                            Prefer using a separate clean browser profile - extensions and non-default browser settings
```

Comment:
super nit: please use a larger dash to separate the 2 parts of the sentence.
According to https://www.scribbr.com/language-rules/dashes/, this is either a `endash` with spaces, or a `emdash` without spaces, depending on whether we want to look british or american :-)